app/src/main/java/com/cmput301w23t40/capturetheqr/QRDetailsActivity.java

```diff
@@ -46,7 +46,9 @@ protected void onCreate(Bundle savedInstanceState) {
         visText.setText(code.getVisualization());
         nameText.setText(code.getCodeName());
         scoreText.setText(String.valueOf(code.getScore()));
-        locationText.setText(code.getGeolocation().toString());
+        if(code.getGeolocation() != null){
```

Comment:
we never specified this, but do we want to display something if there was no recorded geolocation?
e.g. "No geolocation recorded for this Code"

Reply:
yes, you are right. I will make the change right away